Library/Homebrew/test/dev-cmd/bottle_spec.rb

```diff
@@ -194,3 +194,102 @@ def stub_hash(parameters)
     )
   end
 end
+
+describe "brew bottle --merge", :integration_test, :needs_linux do
+  it "adds the bottle block to a formula that has none" do
+    core_tap = CoreTap.new
+    core_tap.path.cd do
+      system "git", "init"
+      setup_test_formula "testball"
+      system "git", "add", "--all"
+      system "git", "commit", "-m", "testball 0.1"
+    end
+
+    File.open("testball-1.0.big_sur.bottle.json", "w") do |f|
+      f.write(
+        stub_hash(
+          {
+            "name":           "testball",
+            "version":        "1.0",
+            "path":           "#{core_tap.path}/Formula/testball.rb",
+            "cellar":         "any_skip_relocation",
+            "os":             "big_sur",
+            "filename":       "hello-1.0.big_sur.bottle.tar.gz",
+            "local_filename": "hello--1.0.big_sur.bottle.tar.gz",
+            "sha256":         "a0af7dcbb5c83f6f3f7ecd507c2d352c1a018f894d51ad241ce8492fa598010f",
+          },
+        ),
+      )
+    end
+
+    File.open("testball-1.0.catalina.bottle.json", "w") do |f|
+      f.write(
+        stub_hash(
+          {
+            "name":           "testball",
+            "version":        "1.0",
+            "path":           "#{core_tap.path}/Formula/testball.rb",
+            "cellar":         "any_skip_relocation",
+            "os":             "catalina",
+            "filename":       "testball-1.0.catalina.bottle.tar.gz",
+            "local_filename": "testball--1.0.catalina.bottle.tar.gz",
+            "sha256":         "5334dd344986e46b2aa4f0471cac7b0914bd7de7cb890a34415771788d03f2ac",
+          },
+        ),
+      )
+    end
+
+    expect {
+      brew "bottle",
+           "--merge",
+           "--write",
+           "testball-1.0.big_sur.bottle.json",
+           "testball-1.0.catalina.bottle.json"
+    }.to output(
+      <<~EOS,
```

Comment:
```suggestion
      <<-EOS,
```

and remove all indentation below (which will fix the test failure). Should be easy to reproduce locally.